operations/update.go

```diff
@@ -39,12 +40,17 @@ func Update() cli.Command {
 				Name:  joinFlagNames(forceFlagName, "f"),
 				Usage: "download a new CLI even if the current CLI is not out of date",
 			},
+			cli.BoolFlag{
+				Name:  joinFlagNames(autoUpdateFlagName, "a"),
```

Comment:
Since the config option is called "AutoUpgradeCLI", can this also be "autoUpgrade" for consistency?